tests/integration/serializers/rest-serializer/serialize-ids-test.js

```diff
@@ -0,0 +1,54 @@
+import { module, test } from 'qunit';
+import Schema from 'ember-cli-mirage/orm/schema';
+import Db from 'ember-cli-mirage/db';
+import SerializerRegistry from 'ember-cli-mirage/serializer-registry';
+import { RestSerializer, Model, hasMany } from 'ember-cli-mirage';
+
+module('Integration | Serializers | Rest Serializer | Serialize ids', function(hooks) {
+  hooks.beforeEach(function() {
+    this.schema = new Schema(new Db(), {
+      wordSmith: Model.extend({
+        blogPosts: hasMany(),
+        specialPosts: hasMany('blog-post', { inverse: 'specialAuthor' })
+      }),
+      blogPost: Model
+    });
+  });
+
+  hooks.afterEach(function() {
+    this.schema.db.emptyData();
+  });
+
+  test(`if serializeIds is 'always' it serailizes the ides of all hasMany associations`, function(assert) {
+    let ApplicationSerializer = RestSerializer;
+
+    let registry = new SerializerRegistry(this.schema, {
+      application: ApplicationSerializer,
+      wordSmith: ApplicationSerializer.extend({
+        serializeIds: 'always'
+      })
+    });
+
+    let wordSmith = this.schema.wordSmiths.create({
+      id: 1,
+      name: 'Link'
+    });
+    wordSmith.createBlogPost();
+    wordSmith.createBlogPost();
+    wordSmith.createSpecialPost();
+
+    let result = registry.serialize(wordSmith);
+
+    assert.deepEqual(result, {
```

Comment:
The test is kind of pointless because I'm just saying this is what Ember Data expects, but no way to confirm this...